src/renderer.js

```diff
@@ -3,6 +3,7 @@ var cartodb = global.cartodb || {};
 var carto = global.carto || require('carto');
 var _ = global._ || require('underscore');
 var geo = require("./geo");
+var Filter = require("./filter");
```

Comment:
The layer (`CartoDBd3Layer`) receives a geojson tile that contains/can contain several layers, each one of which is represented by a renderer. Keeping that in mind, would it be better to create a new class, called something along the lines of `sublayer`, and restrict the functions of the renderer to generating the svgs?